docs/cli/create-graphback.md

```diff
@@ -0,0 +1,73 @@
+---
+id: create-graphback
+title: Create Graphback
+sidebar_label: Create Graphback
+---
+
+[Create Graphback](https://www.npmjs.com/package/create-graphback) is a command-line utility for creating Graphback applications.
+
+
+## Installation
+
+:::note
+This part can be skipped if you do not plan to install the CLI globally. 
```

Comment:
I would skip that part completely. Things like create react app etc. should be never installed globally as they outdated fast